mne/forward/forward.py

```diff
@@ -875,6 +875,70 @@ def write_forward_meas_info(fid, info):
     end_block(fid, FIFF.FIFFB_MNE_PARENT_MEAS_FILE)
 
 
+def _check_loose(forward, loose):
+    """Check loose parameter input."""
+    if is_fixed_orient(forward):
```

Comment:
does it mean I always have a warning when working with fixed ori?

I think we should not have a warning when user set loose to None explicitly.